include/epoxy/gl.h

```diff
@@ -92,6 +92,11 @@ EPOXY_PUBLIC bool epoxy_has_gl_extension(const char *extension);
 EPOXY_PUBLIC bool epoxy_is_desktop_gl(void);
 EPOXY_PUBLIC int epoxy_gl_version(void);
 
+typedef void * (*epoxy_resolver_failure_handler_t)(const char *name);
```

Comment:
I'm not entirely sure why `epoxy_resolver_failure_handler_t` would need to return something.

I mean, if you plan on killing the connection then just calling this function and returning `NULL` from Epoxy itself would be enough; or you do plan on returning some other blob of data?